src/go/rpk/pkg/cli/cmd/redpanda/admin/admin.go

```diff
@@ -60,6 +61,11 @@ func NewCommand(fs afero.Fs) *cobra.Command {
 		&adminCAFile,
 	)
 
+	common.AddBrokersFlag(
```

Comment:
All admin commands operate on the admin port, vs. brokers being for the kafka protocol. Why add this flag?

Reply:
Removed Brokers Flag, I might have been testing with this flag.